astropy/coordinates/tests/test_frames.py

```diff
@@ -625,8 +626,17 @@ def test_representation():
     icrs.representation_type = 'cylindrical'
 
     assert icrs.representation_type is r.CylindricalRepresentation
+    assert icrs_cyl.rho == icrs.rho
+    assert icrs_cyl.phi == icrs.phi
+    assert icrs_cyl.z == icrs.z
     assert icrs.data == data
 
+    # Testing that an ICRS object in SphericalRepresentation must not have spherical attributes.
```

Comment:
I guess it should say `CylindricalRepresentation` here?

Reply:
Thanks for catching that!